tenant-cd/src/main/java/ai/vespa/hosted/cd/Search.java

```diff
@@ -9,6 +9,14 @@
  */
 public class Search {
 
+    private final String raw;
+
+    public Search(String raw) {
```

Comment:
raw... output!? I'm confused about the class name (Search), and what "raw" is supposed to mean. Is this class supposed to represent the result of a search?